OutOfSchool/OutOfSchool.BusinessLogic/Extensions/SearchResultExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OutOfSchool.BusinessLogic.Models;
+
+namespace OutOfSchool.BusinessLogic.Extensions;
+public static class SearchResultExtensions
+{
+    /// <summary>
+    /// Method, that checks SearchResult for null and his entities for empty.
+    /// </summary>
+    /// <typeparam name="T">Typeof entity.</typeparam>
+    /// <param name="searchResult">Instance of SearchResult.</param>
+    /// <returns>
+    /// True - searchResult is null or searchResult.Entities is empty.
+    /// False - searchResult is not null or searchResult.Entities is not empty.
```

Comment:
(Comment wording) Please change comment or drop it. It says `False - searchResult is not null or searchResult.Entities is not empty.` but should be `...not null AND ... is not empty` - "and" instead of "or". 

In general - I don't see necessity to explain such code in comments. Current method name is self-explanatory, IMHO.

Reply:
Comment - deleted.